modules/kuberay-cluster/values.yaml

```diff
@@ -90,16 +90,16 @@ head:
   # for further guidance.
   resources:
     limits:
-      cpu: "4"
+      cpu: "8"
```

Comment:
The worker node uses similar limits. I wonder if the variation in job time was based on whether the task lands on head node or worker node